### DISCUSSION TITLE
Renovate throwing  error

### DISCUSSION THREAD

**Discussion by Rahul-learn** [renovatebot/renovate#9806] [Request Help]
We have self hosted renovate, We are getting below error while executing. Could you please shed some light ?

Error committing files (repository=***/***)
       "err": {
         "task": {
           "commands": [
             "push",
             "origin",
             "renovate/***/**",
             "--force",
             "-u",
             "--no-verify",
             "--verbose",
             "--porcelain"
           ],
           "format": "utf-8"
         },
         "message": "Pushing to https://github.***.git\nPOST git-receive-pack (1097 bytes)\nremote: error: GH003: Sorry, force-pushing to renovate/**** is not allowed.        \nerror: failed to push some refs to 'https://github.***.git'\n",
         "stack": "Error: Pushing to https://github.***.git\nPOST git-receive-pack (1097 bytes)\nremote: error: GH003: Sorry, force-pushing to renovate/com.**** is not allowed.        \nerror: failed to push some refs to 'https://github.***.git'\n\n    at GitExecutorChain.onFatalException (/usr/src/app/node_modules/simple-git/src/lib/runners/git-executor-chain.ts:67:77)\n    at GitExecutorChain.<anonymous> (/usr/src/app/node_modules/simple-git/src/lib/runners/git-executor-chain.ts:59:21)\n    at Generator.throw (<anonymous>)\n    at rejected (/usr/src/app/node_modules/simple-git/src/lib/runners/git-executor-chain.js:6:65)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (internal/process/task_queues.js:93:5)"
       }

**Comment by viceice** [comment]
Please format your logs.

Your branch protections on gitlab are blocking renovate. You should disable them for the renovate branch prefix ( `renovate/` by default)

**Comment by Rahul-learn** [comment]
@viceice 

We don't have any branch protection rule for this repo.

**Comment by viceice** [comment]
GH003 means force push is disallowed by server. So your github enterprise server is blocking it.
